### PR TITLE
(FM-7792) Clean up ntp tests

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

**Comment by eimlav** [comment]
Adhoc ran successfully:

screen shot 2019-02-21 at 14 31 22

File: spec/acceptance/ntp_install_spec.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 require 'spec_helper_acceptance'
 
-case fact('osfamily')
-when 'FreeBSD'
+case os[:family]
+when 'freebsd'
```
Comment (HelenCampbell) [inline spec/acceptance/ntp_install_spec.rb]: I would say you could remove both this and AIX? Since they aren't supported for ntp?

**Comment by tphoney** [comment]
Great work @eimlav

File: spec/acceptance/ntp_install_spec.rb (hunk 1 above)
Comment (eimlav) [reply spec/acceptance/ntp_install_spec.rb]: Will remove FreeBSD occurrences in another PR.

**Pull request merged**